checks.d/etcd.py

```diff
@@ -88,6 +76,20 @@ def check(self, instance):
                 else:
                     self.log.warn("Missing key {} in stats.".format(key))
 
+        storeResponse = self.get_store_metrics(url, timeout)
```

Comment:
This is very nitpicky, but if you could change those names @degemer with python-compliant names (snake_case) I would be :heart_eyes_cat: